app/backend/db/schemas.py

```diff
@@ -0,0 +1,86 @@
+from enum import Enum 
+from typing import List, Dict, Any
+from datetime import date, time, datetime
+
+from pydantic import BaseModel
+
+
+class Mador(BaseModel):
+    name: str
+    manager: int = None
+
+    class Config:
+        orm_mode = True
+
+
+class User(BaseModel):
+    id: int
+    commander: int = None
+    reminder_time: time = None
+    english_name: str = None
+    mador: Mador = None
+    operators_id: List[int] = None
+
+    class Config:
+        orm_mode = True
+
+
+class Hierarchy(BaseModel):
+    """"Hierarchy for User and Commander."""
+    hierarchy: Dict[User, Any] = None
```

Comment:
default should be an empty dict.
also it doesn't match the hierarchy i asked from u

Reply:
I'll change it in this task:
https://trello.com/c/i2qiSqjc/15-add-backend-router-for-users

Reply:
Done